src/constants/index.ts

```diff
@@ -1,12 +1,14 @@
 export { default as ANIMATION } from './animation';
 export { default as BOX_SHADOWS } from './boxShadows';
 export { default as BREAKPOINTS } from './breakpoints';
-export { default as COLORS, COLORS_PROP_TYPES } from './colors';
+// TODO: Remove once all consumer usage is using theming
+export { default as COLORS } from './colors';
 export { default as FONTS } from './fonts';
 export { default as GRID } from './grid';
 export { default as LAYOUT } from './layout';
 export { default as MEDIA_QUERIES } from './mediaQueries';
 export { default as PROGRESS_BAR_STATUS } from './progressBarStatus';
 export { default as SPACER } from './spacer';
+export { primaryTheme, secondaryTheme, COLORS_PROP_TYPES } from './themes';
```

Comment:
Consumer apps do not currently use `COLORS_PROP_TYPES` but we make it available in case we create custom components around it and want the `prop-types` checking. We can remove this later if it's unnecessary.